Update links

TeraDonawa opened the pull request:
Make any link referenced is described relative to the certification document i.e. explains how a student would learn this information using the certification document.

Refine questions using the guide

ashankla committed:
Update README.md

Added links till 21st Question. Committing to save progress due to HP Anyware Site down.

ashankla committed:
Update README.md

1. Updated links from 21st to 57th question
2. Added relevant info to GitHub cert document

smyadari committed:
Update README.md

Updated the admin guide links from version 2024.07 to 2024.10, up to the PCoIP Session Planning Guide

smyadari committed:
Update README.md

Updated admin guide links to the latest 2024.10 version, up to Anyware Client Download link (under Component Links section)

TeraDonawa commented on README.md:
```diff
@@ -324,15 +352,30 @@ model to choose.**
 <img src="media/image1.png" style="width:5.59375in;height:4.39583in" />
 
+
+**Viewing License Information**
```
Should be ### Viewing Licence Information

ashankla committed:
Update README.md

Added viewing Licensing Information section

iroberts405 committed:
Update README.md

added pre-session and session section

iroberts405 commented on README.md:
```diff
@@ -324,15 +352,30 @@ model to choose.**
 
+
+### Viewing License Information
+To view the status of the licenses, use the **_pcoip-list-licenses_** command. 
```
Needs context - the pcoip-list-licenses command exists for LLS and Mac/Linux Agent but Windows Agent is a Powershell script.

smyadari committed:
Update README.md

Updated all the links to the latest 24.10 version

ashankla committed:
Update README.md

Removed extra info from Client Section

ashankla committed:
Update README.md

1. Removed AWM On-Prem section from Component links
2. Modified AWM SaaS info section & links
3. Modified Anyware Connector section to add answers for 2 questions

ashankla committed:
Update README.md

Compressed info under Licensing section

iroberts405 committed:
Update README.md

removing Manager on premises references
